src/Exception.php

```diff
@@ -33,36 +33,48 @@ public static function invalidType(string $expect, $actual) : Exception
         return new Exception("Expected type $expect; got $actual instead.");
     }
 
-    public static function unexpectedRowCountAffected($count)
+    public static function unexpectedRowCountAffected($count) : Exception
```

Comment:
The class is not final but the method declaration is changed. Are you planning a new major release or what does your backwards compatibility promise say in this case?